.github/workflows/push.yaml

```diff
@@ -53,3 +53,43 @@ jobs:
         with:
           name: redskyctl_linux_amd64
           path: dist/redskyctl-linux-amd64.tar.gz
+      - name: KinD (Kubernetes in Docker) Action
+        uses: engineerd/setup-kind@v0.3.0
+      - name: commands
+        run: |
+          kubectl get nodes -o wide
+          kubectl get node kind-control-plane -o yaml
+          kubectl get pods -n kube-system
+      - name: redsky quickstart
+        run: |
+          echo "Install redskyctl"
+          curl -sLo - https://github.com/redskyops/redskyops-controller/releases/download/v1.5.0/redskyctl-linux-amd64.tar.gz | \
+            tar -zxf - && \
+            chmod 755 ./redskyctl
```

Comment:
Was this necessary? The permissions should be preserved.

Reply:
Should be able to drop this and use the precompiled version ( under `dist` ).